src/main.c

```diff
@@ -73,6 +73,9 @@ main() {
   int nfds;
   int udevfd;
 
+  /* init libusb */
+  usb_init();
```

Comment:
usb_init() is defined in usb.h, part of the libusb library (specifically libusb-compat-0.1). usb_init() has to be called once before you can use any of libusb's functionality. Since vusb-daemon already includes usb.h (in project.h) and depends on libusb-compat, you are right to suspect that it may be superfluous here. However, vusb-daemon totally crashed when I tried to remove it...